core/src/main/java/io/keen/client/java/ScopedKeys.java

```diff
@@ -63,7 +63,11 @@
      */
     public static String encrypt(String apiKey, Map<String, Object> options)
             throws ScopedKeyException {
-        return encrypt(KeenClient.client(), apiKey, options);
+        if (apiKey.length() == 64) {
+            return encrypt(KeenClient.client(), apiKey, options);
+        } else {
+            return oldEncrypt(KeenClient.client(), apiKey, options);
```

Comment:
Ah, yeah sorry I didn't notice that the one taking a `KeenClient` was also `public`. Makes sense.